### DISCUSSION TITLE
Tsynth looping through parameters

### DISCUSSION THREAD

**Discussion by Jayromoy** [ElectroTechnique/electrotechnique.github.io#40] [Software Issues]
Hello everybody!!
First, english is not my mother tongue so please excuse any errors on my part.  I am building TSynth. Now I have teensy and audio module soldered. I have connected the lcd display, only this and nothing more. I have flashed the last firmware but when I connect the USB to the teensy, the display is going crazy, and seems that is looping through parameters. I have read that the problem could be the horizontal resolution. I've changed it to 159 but nothing changed. The same with quantise factor to 63. As a last try, I've flashed all old firmwares, one by one. The same problem with all. I've tested the soldering and it's all ok. I have attached a little video. Any idea of what is happening or what can I do? Thanks in advance.

https://user-images.githubusercontent.com/83035347/115772946-ffbb4780-a3af-11eb-817c-a9d0bed5ce14.mp4

**Comment by Electrotechnique** [comment]
The display looks fine except it needs the colours inverted. The potentiometers have noise on them and the multiplexer is reading them rapidly. Each time the value is different, so the parameter is changed and the display will show the new parameter value. It's a hardware problem.

**Reply by Jayromoy** [reply]
But the thing is that the teensy is connected only to the audio board and LCD. No pots, no multiplexer, no encoder, no pcb... Nothing. Only teensy, audio board and LCD is connected.

**Reply by Electrotechnique** [reply]
Which is why it isn't working, all the connections to the multiplexer are floating and need the pots soldered. This is mentioned in the build guide.
